src/Schema/Builder.php

```diff
@@ -165,6 +166,8 @@ public function dropAllTables()
             $blueprint->drop();
             array_push($queries, ...$blueprint->toSql($connection, $this->grammar));
         }
-        $connection->runDdlBatch($queries);
+
+        if(count($queries))
+            $connection->runDdlBatch($queries);
```

Comment:
These fixes don't belong in this PR.

Reply:
ye fair i merged from the other drop-all branch into this